core/src/test/scala/org/apache/spark/sql/MysqlDataTypeSuite.scala

```diff
@@ -1,4 +1,5 @@
 /*
+ *
```

Comment:
should we add `license check` in scalafmt?

Reply:
sure.